src/modules/pmc/src/lib/Pmc.h

```diff
@@ -2,10 +2,13 @@
 // Licensed under the MIT License.
 
 #include <PmcBase.h>
+#include <string>
 
 class Pmc : public PmcBase
 {
 public:
     Pmc(unsigned int maxPayloadSizeBytes);
     ~Pmc() = default;
+
+    int RunCommand(const char* command, bool replaceEol, std::string* textResult, unsigned int timeoutSeconds) override;
```

Comment:
>bool replaceEol, [](http://example.com/codeflow?start=40&length=16)

Do you really need to use this? For all internal commands this could be hardcoded